.github/workflows/build.yml

```diff
@@ -0,0 +1,32 @@
+name: Project CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x, 16.x, 'lts/*']
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v2
+      with:
+        node-version: ${{ matrix.node-version }}
+        cache: 'npm'
+    - name: Install dependencies
+      run: npm install
+    - name: Run tests
+      run: npm test
+    - name: Code coverage
+      run: npm run codecov
+      # codecov to only run in case of push to master
```

Comment:
The existing behavior was that every pull request triggered codecov which resulted in helpful comments like [this](https://github.com/tarunbatra/password-validator/pull/54#issuecomment-724482225). I would like to preserve that feature, unless you disagree.